modules/apps/27-interchain-accounts/keeper/handshake_test.go

```diff
@@ -214,19 +220,23 @@ func (suite *KeeperTestSuite) TestOnChanOpenAck() {
 
 			err = path.EndpointB.ChanOpenTry()
 			suite.Require().NoError(err)
+			expectedChannel = path.EndpointA.ChannelID
 
 			tc.malleate() // explicitly change fields in channel and testChannel
 
 			err = suite.chainA.GetSimApp().ICAKeeper.OnChanOpenAck(suite.chainA.GetContext(),
 				path.EndpointA.ChannelConfig.PortID, path.EndpointA.ChannelID, counterpartyVersion,
 			)
 
+			activeChannel, _ := suite.chainA.GetSimApp().ICAKeeper.GetActiveChannel(suite.chainA.GetContext(), path.EndpointA.ChannelConfig.PortID)
+
+			suite.Require().Equal(activeChannel, expectedChannel)
```

Comment:
maybe in tc.expPass this should be expected Channel and in the else it should be an empty string?

Reply:
Ah sorry, this was on auto-merge :D

Reply:
Yeah, your suggestion prob reduces the need to copy pasta `expectedChannel = ""` in the malleate functions. I'll add it into another PR.